views/digitaljs/src/main.ts

```diff
@@ -22,6 +22,25 @@ interface MessageDocument {
 
 type Message = MessageDocument;
 
+function getSvg(svgElem: Element, width: Number, height: Number): String {
```

Comment:
Please don't use the `function` keyword and instead use arrow functions. Additionally, the TypeScript types for number and string should be lowercase.

```suggestion
const getSvg = (svgElem: Element, width: number, height: number): string => {
```